routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(_req, res, _next) {
+  res.send('respond with a resource');
+});
+
+module.exports = router;
```

Comment:
Given we don't have users in place at the moment and will likely be using cognito for access management I would suggest we take this out and only add it back in if it is required